packages/@react-aria/utils/src/useFormReset.ts

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
```

Comment:
I put this in `@react-aria/utils` for now. Wondering if we should make a new package like `@react-aria/form` for it though. I thought utils since it wasn't really meant as public API, but there is a lot here. :shrug:

Reply:
yes, i think it should be in a form package, even if it's not public right now, we may want it to be eventually. we do that in other places, we just don't document the things in it and we don't mention the package on our docs website.
If we wanted, we could start putting in the Readme that these are intended as private as well